lib/projects/next-prototype-builder/templates/app/page.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export default function Page() {
+  return (
+    <main>
+      <h1>Welcome to {{projectTitle}}</h1>
+      <p>{{projectDescription}}</p>
```

Comment:
This is what fails if projectDescription is empty

Reply:
Cool, thanks. I'll verify the input.

Reply:
I've pushed a fix: added default project description based on project type.

Also changed prototype selection question to: "What type of project prototype would you like to create?"